modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/list-type-definitions/request-status.json

```
@@ -0,0 +1,6 @@
+{
+	"name": "RequestStatus",
+	"name_i18n": {
+		"en-US": "Request Status"
```

Following your naming pattern it should be: 
```suggestion
		"en-US": "Request: Request Status"
```